### PR TITLE
Adjust telegraph configuration

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Current configuration may lead to high CPU and MEM usage.
This changes will reduce used resources with slightly reducing
intervals and increasing buffer sizes. This should not reduce metrics
granularity but will only change metrics refresh interval.

## Corresponding DC/OS tickets (required)

  - [COPS-5629](https://jira.d2iq.com/browse/COPS-5629) Telegraf saturates CPU on agents

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary @pierrebeitz

File: CHANGES.md [hunk 1]
```diff
@@ -26,3 +26,5 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Ensured that marathon and SDK labeled reservations are not offered to other schedulers (D2IQ-68800)
 
 * Updated OpenResty to 1.15.8.4. (DCOS_OSS-5967)
+
+* Update Telegrapf configuration to reduce used resources on statsd inputs. (COPS-5629)
```
Comment (jongiddy) [inline CHANGES.md:30]: spelling is Telegraf

`reduce used resources on statsd inputs` - for the release notes, can we relate this more to user experience (e.g. reduce errors, vary requests to reduce load, sample less frequently)

**Comment by janisz** [comment]
@jongiddy I fixed typos

**Comment by janisz** [comment]
@mesosphere-mergebot backport 2.1 2.0 1.13

**Pull request merged**